TextInputKit/TextInputKit/Code/AppKitExtensions/NSText+TextInputKit.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  NSText+TextInputKit.swift
+//  TextInputKit
+//
+//  Created by Artem Starosvetskiy on 19/12/2016.
+//  Copyright © 2016 Artem Starosvetskiy. All rights reserved.
+//
+
+import AppKit
+
+extension NSText {
+
+    var textInputKit_selectedUtf16IntRange: Range<Int>? {
```

Comment:
Just a question: why don't you use a shortened prefix for framework category methods?